graph/registry_credential.go

```diff
@@ -15,7 +15,7 @@ var (
 	errInvalidUsername      = errors.New("username can't be empty")
 	errInvalidPassword      = errors.New("password can't be empty")
 	errInvalidIdentity      = errors.New("identity can't be empty")
-	errInvalidArmResourceID = errors.New("armResourceId can't be empty")
```

Comment:
Recommend renaming this to `resourceId` ref  https://github.com/Azure/azure-sdk-for-go/blob/1951233eb944a49aa5f7278cba8e3e32a8c958af/services/resources/mgmt/2018-05-01/resources/resources.go#L137